src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -246,54 +246,46 @@ export const ControlGroupStrings = {
+        }),
+      validateSelections: {
+        getValidateSelectionsTitle: () =>
+          i18n.translate('controls.controlGroup.management.validate.title', {
+            defaultMessage: 'Validate user selections',
+          }),
+        getValidateSelectionsSubTitle: () =>
+          i18n.translate('controls.controlGroup.management.validate.subtitle', {
+            defaultMessage:
+              'Automatically ignore any control selection that would result in no data.',
+          }),
+      },
+      controlChaining: {
+        getHierarchyTitle: () =>
+          i18n.translate('controls.controlGroup.management.hierarchy.title', {
+            defaultMessage: 'Chain controls',
+          }),
+        getHierarchySubTitle: () =>
+          i18n.translate('controls.controlGroup.management.hierarchy.subtitle', {
+            defaultMessage:
+              'Selections in one control narrow down available options in the next. Controls are chained from left to right.',
+          }),
+      },
     },
-    /** TODO: These translations aren't used but they will be once https://github.com/elastic/kibana/issues/162985 is resolved */
-    querySync: {
-      getQuerySettingsTitle: () =>
-        i18n.translate('controls.controlGroup.management.query.searchSettingsTitle', {
-          defaultMessage: 'Sync with query bar',
-        }),
-      getQuerySettingsSubtitle: () =>
-        i18n.translate('controls.controlGroup.management.query.useAllSearchSettingsTitle', {
-          defaultMessage:
-            'Keeps the control group in sync with the query bar by applying time range, filter pills, and queries from the query bar',
-        }),
-      getAdvancedSettingsTitle: () =>
-        i18n.translate('controls.controlGroup.management.query.advancedSettings', {
-          defaultMessage: 'Advanced',
-        }),
-      getIgnoreTimerangeTitle: () =>
-        i18n.translate('controls.controlGroup.management.query.ignoreTimerange', {
-          defaultMessage: 'Ignore timerange',
-        }),
-      getIgnoreQueryTitle: () =>
-        i18n.translate('controls.controlGroup.management.query.ignoreQuery', {
-          defaultMessage: 'Ignore query bar',
-        }),
-      getIgnoreFilterPillsTitle: () =>
-        i18n.translate('controls.controlGroup.management.query.ignoreFilterPills', {
-          defaultMessage: 'Ignore filter pills',
+    filteringSettings: {
+      getFilteringSettingsTitle: () =>
+        i18n.translate('controls.controlGroup.management.filteringSettings', {
+          defaultMessage: 'Filtering',
+        }),
+      getUseGlobalFiltersTitle: () =>
+        i18n.translate('controls.controlGroup.management.filtering.useGlobalFilters', {
+          defaultMessage: 'Apply global filters to controls',
+        }),
+      getUseGlobalTimeRangeTitle: () =>
+        i18n.translate('controls.controlGroup.management.filtering.useGlobalTimeRange', {
+          defaultMessage: 'Apply global time range to controls',
```

Comment:
Unfortunately, there is very little consistency in the copy different apps use for their "unified search bar" setting toggles:

| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; App &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Screenshot |
|--------|--------|
| Maps | <img width="400" src="https://github.com/elastic/kibana/assets/8698078/33c082de-7b23-4a2d-bc05-264a01326938"/> |
| Lens | <img width="400" src="https://github.com/elastic/kibana/assets/8698078/9f9062f5-b177-452e-a5e6-b60528dcb542"/> |
| TSVB | <img width="400" src="https://github.com/elastic/kibana/assets/8698078/db1058fe-6037-4d65-bf70-9d488fefc24c"/> | 

So, since there didn't seem to be universal phrasing, I decided that the Maps copy is the best match in this case and went with that - open to suggestions, though 🤔 

![Screenshot 2023-12-18 at 1 34 06 PM](https://github.com/elastic/kibana/assets/8698078/1765f18e-c377-4db7-853b-d21b44f89a4d)

Reply:
cc @amyjtechwriter for the copy here :eyes: